.pipelines/scripts/run-e2e.sh

```diff
@@ -145,6 +146,6 @@ fi
 
 export E2E_ON_AKS_CLUSTER=true
 if [[ "${CLUSTER_TYPE:-}" =~ "autoscaling" ]]; then
-  export LABEL_FILTER="Feature:Autoscaling || !Serial && !Slow"
```

Comment:
Here autoscaling tests for cloud-provider-azure and cluster-autoscaler are different, so you can have your own `LABEL_FILTER` without touching the current one.
Suggestion: In your pipeline, set `LABEL_FILTER` and add a if statement here to check if `LABEL_FILTER` has already been set.

Reply:
I have changed this so LABEL_FILTER is checked for null or empty. (Will change to "Feature:Autoscaling || !Serial && !Slow" on default)